eqllib/analytics/collection/T1005-txt-files-with-command-line-redirection.toml

```
@@ -0,0 +1,22 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies Koadic's use of redirecting command output to plaintext files."
```

In general, we've tried to steer clear of tool-focused rules, and emphasize behaviors that are shared. I think this technique is actually very generic, so it would be better to articulate that it detects much more than Koadic. We can add details about why this is useful to an attacker: 

```rest

```
```suggestion
description = """
Identifies command execution via WMI with redirected output. WMI provides a method to execute a process on a local or remote host, but does not expose a way to read any console output. To get around this restriction, some administrators or attackers will execute ``cmd.exe`` with output redirection to a file. Then the file can be retrieved to read program output.

When this happens, the process tree will resemble this:
.. code-block::

    * svchost.exe
     `-- wmiprvse.exe
        `-- cmd.exe systeminfo.exe > C:\wmi-temp.log
             `-- systeminfo.exe
```